index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Linus Shaddai Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img src="./img/Logo Placeholder.svg" alt="logo" />
+        <img class="hamburger-menu" src="./img/Union.svg" alt="hambuger menu" />
+      </nav>
```

Comment:
Kindly make sure the `img` Logo is wrapped inside an anchor tag `<a href="#"></a>` as it's supposed to be a link that redirects to the home page.